integration-tests/pom.xml

```diff
@@ -18,11 +18,11 @@
     <module>simple-ollama</module>
     <module>azure-openai</module>
     <module>devui</module>
-    <module>in-process-embedding-all-minilm-l6-v2-q</module>
-    <module>in-process-embedding-all-minilm-l6-v2</module>
-    <module>in-process-embedding-bge-small-en-q</module>
-    <module>in-process-embedding-bge-small-en</module>
-    <module>in-process-embedding-e5-small-v2-q</module>
-    <module>in-process-embedding-e5-small-v2</module>
+    <module>embed-all-minilm-l6-v2-q</module>
```

Comment:
Why changing the module names?

Reply:
Just to match the common conventions. Is there a special reason to keep these special?

Reply:
We used the names that are upsteam langchain4j uses, but since this is just about ITs, I don't think it matters much

Reply:
yes, I think it's fine in this case.